src/lxml/tests/test_htmlparser.py

```diff
@@ -592,6 +592,13 @@ def test_ietf_decl(self):
         self.assertEqual(self.etree.tostring(doc),
                          _bytes('<!DOCTYPE html PUBLIC "-//IETF//DTD HTML//EN">\n<html/>'))
 
+    def test_boolean_attribute(self):
+        # ability to serialize boolean attribute by setting value to None
+        form = html.Element('form')
+        form.set('novalidate', None)
+        self.assertEqual(html.tostring(form), 
+                _bytes('<form novalidate></form>'))
+
```

Comment:
I know, but you are now making this an official feature. We should better make sure that it actually works, and keeps working.